tests/libpod/blackhole.go

```diff
@@ -0,0 +1,54 @@
+package libpod
+
+import (
+	"context"
+
+	. "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests/exec"
+)
+
+func AddKubernetesApiBlackhole(pods *v1.PodList, containerName string) {
+	serviceBlackHoleForKubernetesApiToPod(pods, containerName, true)
+}
+
+func DeleteKubernetesApiBlackhole(pods *v1.PodList, containerName string) {
+	serviceBlackHoleForKubernetesApiToPod(pods, containerName, false)
+}
+
+func serviceBlackHoleForKubernetesApiToPod(pods *v1.PodList, containerName string, present bool) {
+	var addOrDel string
+
+	virtCli, err := kubecli.GetKubevirtClient()
+	Expect(err).NotTo(HaveOccurred())
+
+	serviceIp, err := getKubernetesApiServiceIp(virtCli)
+	Expect(err).NotTo(HaveOccurred())
+
+	if present {
+		addOrDel = "add"
+	} else {
+		addOrDel = "del"
+	}
+
+	for _, pod := range pods.Items {
+		_, err = exec.ExecuteCommandOnPod(virtCli, &pod, containerName, []string{"ip", "route", addOrDel, "blackhole", serviceIp})
+		Expect(err).NotTo(HaveOccurred())
+	}
+}
+
+func getKubernetesApiServiceIp(virtClient kubecli.KubevirtClient) (string, error) {
+	kubernetesServiceName := "kubernetes"
+	kubernetesServiceNamespace := "default"
+
+	kubernetesService, err := virtClient.CoreV1().Services(kubernetesServiceNamespace).Get(context.Background(), kubernetesServiceName, v12.GetOptions{})
+	if err != nil {
+		return "", err
+	}
```

Comment:
Use `Expect(err).NotTo(HaveOccurred())` since we are in test code?

Reply:
Done.